packages/shopify-codemod/transforms/remove-empty-returns.js

```diff
@@ -0,0 +1,35 @@
+export default function removeEmptyReturns({source}, {jscodeshift: j}, {printOptions = {}}) {
+  const sourceAST = j(source);
+
+  sourceAST
+  .find(j.FunctionDeclaration, {
```

Comment:
Also, I think we should be combine this into a single transform that just checks that the type is one of the three expressions we care about.